### PR TITLE
Add lesson20

### PR DESCRIPTION
**Pull request by haraldF**

### CODE REVIEW

File: lesson20.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+---
+
+### moment.js example
+
+```html
+<html>
+  <head>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.25.3/moment.js"></script>
+      <script>
```
Comment (sevtapim) [inline lesson20.md]: To prove that we can use the script loaded in header, isn't better to put the code using moment in the body part. To keep the natural flow going and not to wrongly signal that you can only use the scripts loaded in header also in header.

File: lesson20.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+---
+
+### Homework Hints
+
+Let's populate our `<select>` element:
+
+```js
+// inside your fetchCountries() function:
+let countriesElement = document.getElementById("countries");
```
Comment (sevtapim) [inline lesson20.md:247]: Remind that countries is the id we gave to the select HTML element not the variable name we used in the fetchCountries function

File: lesson20.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+
+for (let country of countries) {
```
Comment (sevtapim) [inline lesson20.md:249]: Remind here that countries this time referring to the data we fetched in the function

File: lesson20.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+    let countryElement = document.createElement("option");
+    // put the name of the country as textContent
+    countryElement.textContent = country.name;
+    // put the latitude and longitude as the option's value attribute
+    // since value must be a string, let's convert the latlng array to JSON:
+    countryElement.value = JSON.stringify(country.latlng);
```
Comment (sevtapim) [inline lesson20.md:255]: Explain why we are stringifying the latlng property. Maybe even show the typeof country.latlng before doing the conversion.

**Review by sevtapim — CHANGES_REQUESTED** [review]

**Review by carlotrimarchi — APPROVED** [review]

**Pull request merged**